DuckDuckGo/Preferences/Model/AppearancePreferences.swift

```diff
@@ -117,6 +117,11 @@ final class DefaultHomePageNavigator: HomePageNavigator {
             if let window = WindowControllersManager.shared.lastKeyMainWindowController {
                 let homePageViewController = window.mainViewController.browserTabViewController.homePageViewController
                 homePageViewController?.settingsVisibilityModel.isSettingsVisible = true
+
+                if NSApp.delegateTyped.featureFlagger.isFeatureOn(.htmlNewTabPage) {
```

Comment:
Question about the feature flag: I saw that the `htmlNewTabPage` feature has an `isLaunched` subfeature, but I didn't see it used anywhere yet. If there's any chance that you'll be using the `rollouts` array to roll this out, I'd use that instead, since top level features don't support rollouts. If not, then the current implementation is fine as-is.

(I've been meaning to post about this to the team since this detail bit us for another feature earlier this week.)

Reply:
Yes, I do plan to use the rollouts array and that was the reason for adding a subfeature :) it's just not implemented yet and will come in a separate PR.